redisinsight/api/src/common/decorators/transform-to-map.decorator.ts

```diff
@@ -0,0 +1,15 @@
+import { Transform, plainToClass } from 'class-transformer';
+
+export function TransformToMap<T>(cls: new (...args: any[]) => T) {
+  return Transform(({ value }) => {
```

Comment:
@pawelangelow your tests should pass with:
```suggestion
  return Transform((value) => {
```

Reply:
I mean we must consider bidirectional transformation (to class and to plain). You can find example here: https://github.com/RedisInsight/RedisInsight/blob/b2d53dc903cb78cc4093b6456b64049e06305143/redisinsight/api/src/common/decorators/object-as-map.decorator.ts

Reply:
Thanks for the feedback, both of you! So, here's the [commit](https://github.com/RedisInsight/RedisInsight/pull/4389/commits/fa4b3017779ed4e75ea9dfc6a04d98166fc5b035).

Reply:
@pawelangelow don't you need to include [these](https://github.com/RedisInsight/RedisInsight/blob/b2d53dc903cb78cc4093b6456b64049e06305143/redisinsight/api/src/common/decorators/object-as-map.decorator.ts#L22) transform options as well?

Reply:
Yes! I don't know but kind of expected the transformation relies on order (forward is the first transformation, backwards is the second one). 😄